### ISSUE TITLE
Secondary sinks don't get full telemetry context

### ISSUE THREAD

**Issue by RamjotSingh** [Microsoft/ApplicationInsights-dotnet#992]
## Repro Steps
1. Initialize TelemetryConfiguration with a secondary sink
2. Send a telemetry item through the pipeline

##  Actual Behavior
Telemetry item received by secondary sink does not have properties like TelemetryContext.Operation filled with the details.

## Expected Behavior
TelemetryContext.Operation should be filled with the same details as primary sink.

## Version Info 
SDK Version : 2.8.1
.NET Version : NetStandard2.0					
How Application was onboarded with SDK(VisualStudio/StatusMonitor/Azure Extension) : VS
OS : Windows 10 64Bit
Hosting Info (IIS/Azure WebApps/ etc) : Local InProcess hosting using Kestrel

**Comment by RamjotSingh** [comment]
My guess is this is due to TelemetryContext.DeepClone not copying the required properties over.
```cs
internal TelemetryContext DeepClone(IDictionary<string, string> properties)
        {
            var other = new TelemetryContext(properties);
            // This check avoids accessing the public accessor GlobalProperties
            // unless needed, to avoid the penality of ConcurrentDictionary instantiation.
            if (this.GlobalPropertiesValue != null)
            {
                Utils.CopyDictionary(this.GlobalProperties, other.GlobalProperties);
            }

            if (this.PropertiesValue != null)
            {
#pragma warning disable CS0618 // Type or member is obsolete
                Utils.CopyDictionary(this.Properties, other.Properties);
#pragma warning restore CS0618 // Type or member is obsolete
            }

            other.InstrumentationKey = this.InstrumentationKey;

            // RawObject collection is not cloned by design, they share the same collection.
            other.rawObjectsTemp = this.rawObjectsTemp;
            other.rawObjectsPerm = this.rawObjectsPerm;
            return other;
        }
```
Calling into Initialize() method in the same class should do the trick.

**Comment by cijothomas** [comment]
Fixed and merged. To be released part of 2.9.0-beta1 Thanks @RamjotSingh
